packages/nextjs/app/builders/0x85ba183Fb4c16D435B076DDe459dAfc8eFF864d8/page.tsx

```diff
@@ -0,0 +1,95 @@
+"use client";
```

Comment:
You don't need this line.
You will only use "use client" on components that are using `useEffect`, `useState` or other hooks that manages states and dynamic rendering.